app/services/one_off/fhir/prescription_drug_exporter.rb

```diff
@@ -0,0 +1,118 @@
+require "fhir_models"
+
+module OneOff
+  module Fhir
+    class PrescriptionDrugExporter
+      attr_reader :prescription_drug
+
+      def initialize(prescription_drug)
+        @prescription_drug = prescription_drug
+      end
+
+      def export
+        FHIR::MedicationRequest.new(
+          contained: [
+            FHIR::Medication.new(
+              identifier: FHIR::Identifier.new(
+                value: prescription_drug.id
+              ),
+              code: FHIR::CodeableConcept.new(
+                coding: FHIR::Coding.new(
+                  system: "http://www.nlm.nih.gov/research/umls/rxnorm",
+                  code: prescription_drug.rxnorm_code,
+                  display: prescription_drug.name
+                )
+              )
+            )
+          ],
+          status: prescription_drug.is_deleted ? "ended" : "active",
+          intent: "proposal",
+          medicationReference: FHIR::Reference.new(
+            id: FHIR::Medication.new(
+              identifier: FHIR::Identifier.new(
+                value: prescription_drug.id
+              ),
+              code: FHIR::CodeableConcept.new(
+                coding: FHIR::Coding.new(
+                  system: "http://www.nlm.nih.gov/research/umls/rxnorm",
+                  code: prescription_drug.rxnorm_code,
+                  display: prescription_drug.name
+                )
+              )
+            )
+          ),
+          dosageInstruction: dosage_instruction,
+          dispenseRequest: dispense_request,
+          performer: FHIR::Reference.new(
+            id: FHIR::Organization.new(
+              identifier: FHIR::Identifier.new(
+                value: prescription_drug.facility_id
+              )
+            )
+          ),
+          subject: FHIR::Reference.new(
+            id: FHIR::Patient.new(
+              identifier: FHIR::Identifier.new(
+                value: prescription_drug.patient_id
+              )
+            )
+          ),
+          meta: FHIR::Meta.new(
+            lastUpdated: prescription_drug.device_updated_at.iso8601,
+            createdAt: prescription_drug.device_created_at.iso8601
+          )
+        )
+      end
+
+      def dosage_instruction
+        if prescription_drug.frequency.nil?
+          return nil
+        end
```

Comment:
[non-blocking]

```suggestion
        return if prescription_drug.frequency.nil?
```

Reply:
[suggestion] Instead of returning nil if there's no frequency, can we create a `doseAndRate` without the `timing` field? That should be allowed according to the standard.